README.md

```diff
@@ -19,7 +19,7 @@ co-ordinates, and a letter representing one of the four cardinal compass
 points (`N`, `S`, `E`, `W`). 
 
 The plateau is divided up into a grid to simplify navigation. 
-An example position might be `0 0 N`, which means the rover is in 
+An example position might be 0, 0, N, which means the rover is in 
```

Comment:
I'm trying to use the  same syntax for positions that you find in the input file (to avoid confusion), I'm happy to change it

Reply:
I hadn't even spotted that - let's go with yours, I didn't realise I'd changed it.